trunk/SUAVE/Methods/Weights/Buildups/eHelicopter/empty.py

```diff
@@ -0,0 +1,154 @@
+from SUAVE.Attributes import Solids
+from fuselage import fuselage
+from prop import prop
+from wiring import wiring
+import numpy as np
+
+#-------------------------------------------------------------------------------
+# Empty
+#-------------------------------------------------------------------------------
+
+def empty(rProp, mBattery, mMotors, mPayload, MTOW, propBlades, tailBlades, fLength, fWidth, fHeight):
+    """weight = SUAVE.Methods.Weights.Correlations.eHelicopter.empty(
+            rProp,
+            mBattery,
+            mMotors,
+            mPayload,
+            MTOW,
+            propBlades,
+            tailBlades,
+            fLength,
+            fWidth,
+            fHeight
+        )
+
+        Calculates the empty fuselage mass for an electric helicopter including
+        seats, avionics, servomotors, ballistic recovery system, rotor and hub
+        assembly, transmission, and landing gear. Additionally incorporates
+        results of the following correlation scripts:
+
+            fuselage,py
+            prop.py
+            wing.py
+            wiring.py
+
+        Originally written as part of an AA 290 project inteded for trade study
+        of the eHelicotor along with the following defined SUAVE vehicle types:
+
+            eTiltwing
+            eTiltrotor
+            eStopped_Rotor
+
+        Inputs:
+
+            rProp:      Propeller Radius                [m]
+            mBattery:   Battery Mass                    [m]
+            mMotors:    Total Motor Mass                [m]
+            mPayload:   Payload Mass                    [m]
+            MTOW:       Maximum TO Weight               [N]
+            propBlades: Number of Propeller Blades      [Unitless]
+            tailBlades: Number of Tail Rotor Blades     [Unitless]
+            fLength:    Fuselage Length                 [m]
+            fWidth:     Fuselage Width                  [m]
+            fHeight:    Fuselage Height                 [m]
+
+        Outputs:
+
+            weight:     Dictionary of Component Masses  [m]
+
+    """
+
+    weight = {}
+
+#-------------------------------------------------------------------------------
+# Assumed Weights
+#-------------------------------------------------------------------------------
+
+    weight['Payload']       = mPayload
+    weight['Seats']         = 30.
+    weight['Avionics']      = 15.
+    weight['Motors']        = mMotors
+    weight['Battery']       = mBattery
+    weight['Servos']        = 5.2
+    weight['BRS']           = 16.
+    weight['Hub']           = MTOW/9.8 * 0.04
+    weight['Landing Gear']  = MTOW/9.8 * 0.02
+
+#-------------------------------------------------------------------------------
+# Calculated Weights
+#-------------------------------------------------------------------------------
+
+    # Preparatory Calculations
+
+    sound       = 340.294       # Speed of Sound
+    tipMach     = 0.65          # Propeller Tip Mach limit
```

Comment:
Don't hardcode this type of stuff. You need to default them or allow them to be inputs.

Reply:
Changed to function input with default value.